### ISSUE TITLE
Push to backend needs a sanity check before performing

### ISSUE THREAD

**Issue by sudheesh001** [loklak/loklak_server#1540]
- Issue type: Feature request / Enhancement

So this proposal is broken into separate parts which need to be implemented 1 after the other for this to be completed.
1. Improve the response from the `hello.json` to take a value from `config` called `accept-push-messages: true`. If this flag is set to be true, publish the space available in the backend in the response to `hello.json`
2. Caretaker threads currently fail to push via the `push.json` api to the backend servers (main peer) either when the disk runs out space or when the servers are down. Performing a `hello.json` check and finding out disk space before pushing the messages seems to be a good option. This will also ensure that if the server is down at any point, only a failed socket error is thrown, instead of failed message push stack traces.

Some thought experiments:

Can we use a new flag in `config` like `player: junior|senior` above to be published to the main backend peer to register the IP address of the peer in `peers.json`. A local instance running on someones PC could fetch these tables of registered network nodes periodically and push to them in round robin fashion or if a server denies a push for some reason?

**Comment by mariobehling** [comment]
Yes, we had a discussions about a similar implementation earlier. Sounds like it can work well for me.

**Comment by singhpratyush** [comment]
This is dependent on #1460.
